examples/dt/compute/compute-starter-kit/values.yaml

```diff
@@ -0,0 +1,189 @@
+# local-config: referenced, but not emitted by kustomize
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: network-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  # nodes
+  node_0:
+    name: crc
```

Comment:
ok this needs to be updated to match  the name of the CRC VM

for my current instance this should be 

 node_0:
    name: crc-n5gv4-master-0
    internalapi_ip: 172.17.0.5
    tenant_ip: 172.19.0.5
    ctlplane_ip: 192.168.122.19
    storage_ip: 172.18.0.5

that is why the network config was broken
it was not finding a node called crc